front/app/api/permissions/types.ts

```diff
@@ -26,6 +26,7 @@ export interface IGlobalPermissionData {
     global_custom_fields: boolean;
     verification_enabled: boolean;
     verification_expiry: number | null;
+    everyone_tracking_enabled: boolean;
   };
```

Comment:
@jamesspeake I see this is added to both "Global" and "Phase" permissions - what does that mean exactly? What does the "Global" permission do?

Reply:
Yes it won't ever be used on global permission, but the attribute is there in the data that is returned so I added it for completeness